src/main/groovy/com/instabot/data/model/user/User.groovy

```diff
@@ -9,7 +9,7 @@ import java.time.LocalDateTime
 
 // TODO If the name has changed -> change it on user
 // TODO When creating, check if in white list
-
+// TODO Use Lombok
```

Comment:
I tried, but it seems that Lombok doesn't really work with Groovy.
Actually, I can remove standard getters and setters, as Groovy is creating them by default. 
Are there any other Lombok features you had in mind? The chances are there is something similar built-in Groovy that I could find.